department-of-veterans-affairs/vets-website
# 73445 add certification pages to 1010d

## Summary

This PR introduces the first pass at the Certification Information section of IVC/CHAMPVA form 10-10D. The key goal of this addition is to allow users to provide identifying information when certifying the form and its contents. Some unit tests have been added with more planned in a followup PR.

The changes introduced do not impact anywhere other than the specific 10-10D form app.

- Added the certification information pages to the 10-10D config file
- Created a small helper function to manage the statement of truth signature block
- Added a few unit tests to address the statement of truth signature block logic
- Adjusted page numbering within the form config and updated corresponding unit tests to match
- **Team**: IVC-CHAMPVA (responsible for adding form 10-10D)
- **Flipper**: Not in use, form is not set to go to prod

## Related issue(s)

- [73445](https://github.com/department-of-veterans-affairs/va.gov-team/issues/73445)

## Testing done

- Manual testing to verify fields behave as expected
- Added new unit tests addressing statement of truth logic
- Ran `yarn test:coverage-app ivc-champva/10-10D` to verify existing and new tests run and pass

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |        |![mobile1](https://github.com/department-of-veterans-affairs/vets-website/assets/18408628/e31171a0-14a0-4e74-9e80-4b547d1445a5)|
| Desktop |        |![desktop1](https://github.com/department-of-veterans-affairs/vets-website/assets/18408628/1456d429-18ee-4139-815b-5e4f250fb07b)|

## What areas of the site does it impact?

This form only.

## Acceptance criteria

### Quality Assurance & Testing


### Error Handling


### Authentication


### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:


## Requested Feedback

NA
